### ISSUE TITLE
stack ghci does not work

### ISSUE THREAD

**Issue by IvanMalison** [travitch/taffybar#178]
stack ghci fails with the following output:
```
/usr/bin/ld: /home/imalison/Projects/taffybar/.stack-work/dist/x86_64-linux-tinfo6/Cabal-1.22.5.0/build/
taffybar/taffybar-tmp/src/gdk_property_change_wrapper.o: relocation R_X86_64_32 against `.rodata.str1.1'
 can not be used when making a shared object; recompile with -fPIC                                     
/usr/bin/ld: final link failed: Nonrepresentable section on output
collect2: error: ld returned 1 exit status
```

**Comment by IvanMalison** [comment]
This is now fixed as of ccbf7e45f76781827420f5cf8ff26a26da2eba80

**Comment by IvanMalison** [comment]
If you get this error delete the .stack-work directory and try again.
